content/docs/1.8.0/monitoring/metrics.md

```diff
@@ -85,3 +85,34 @@ weight: 3
 | longhorn_backup_backing_image_actual_size_bytes | Actual size of this backup backing image | longhorn_backup_backing_image_actual_size_bytes{backup_backing_image="parrot"} 3.3554432e+07 |
 | longhorn_backup_backing_image_state | State of this backup backing image: 0=New, 1=Pending, 2=InProgress, 3=Completed, 4=Error, 5=Unknown | longhorn_backup_backing_image_state{backup_backing_image="parrot"} 3 |
 
+## CSI
+
+CSI sidecar component has built-in metrics for users to get insights of CSI operations including total count, error count and call latency. Longhorn exposes the service for each csi sidecar to allow users to access these metrics.
```

Comment:
```suggestion
The CSI sidecar component has built-in metrics for users to get insights into CSI operations. The CSI operations metrics cover total count, error count, and call latency. Longhorn exposes the service for each CSI sidecar component to allow users to access these metrics.
```

Reply:
fixed. thanks

Reply:
we have changed a bit of the content, PTAL, thanks